include/sqlpp11/data_types/text/like.h

```diff
@@ -45,7 +45,7 @@ namespace sqlpp
 
     struct _alias_t
     {
-      static constexpr const char _literal[] = "like_";
+      static constexpr const char _literal[6] = "like_";
```

Comment:
Is this really required by the current MSVC? That would be really annoying! I am so not willing to count characters myself...

If it is required, please add a reference to the corresponding issue so that this can be cleaned up again asap.